cucumber/features/devWelcome.feature

```diff
@@ -0,0 +1,5 @@
+Feature: TTA Smarthub Welcome page
+ 
+    Scenario: "Welcome to the TTA Smart Hub!" is displayed on the welcome page
+        When visiting https://tta-smarthub-dev.app.cloud.gov/
+        Then we should see "Welcome to the TTA Smart Hub!"
```

Comment:
Looks like this scenario is duplicated with one defined in `devLoginPup.feature`